Basic_Resistor_Sensor_Reading_on_Raspberry_Pi/Basic_Resistor_Sensor_Reading_on_Raspberry_Pi.py

```diff
@@ -0,0 +1,29 @@
+# Example for RC timing reading for Raspberry Pi
+# using CircuitPython Libraries
+
+import time
+import board
+from digitalio import DigitalInOut, Direction
+
+def RCtime (RCpin):
+    reading = 0
+
+    # setup pin as output and direction low value
+    rc = DigitalInOut(RCpin)
```

Comment:
change this whole function to a `with DigitalInOut(RCpin) as rc:` so it gets deinitialized when the function's over

Reply:
@ladyada - I made the requested changes to drop the extra initialization and use with instead of the RCpin function. The code is failing on another part of the build @CollinCunningham Neotrellis_M4_Live_Launcher is getting stuck on the "sam.h" header not being available. How would you like me to proceed?

Reply:
ive reset the travis to rerun

Reply:
@ladyada Thank you. If you are comfortable with the coding style, please approve this PR so I can complete the guide tomorrow.